packages/release-mastermind/src/action.ts

```diff
@@ -312,13 +342,28 @@
       ctx = new Issues(this.util, runners, config, curContext, this.dryRun)
       ctx.run().catch(err => {
         throw log(
-          LoggingLevels.error, `Error thrown while running context: `, err)
+          LoggingLevels.error,
+          `Error thrown while running context: `,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
          "Error thrown while running context: ",
```